Persistant-Storage-Snapshot/Threader.java

```diff
@@ -0,0 +1,9 @@
+// Where should this file go?
+
+public class Threader {
+	public static void main(String[] args) {
+        Snapshotter snap = new Snapshotter();
+        Thread thread = new Thread(snap);
+        thread.start();
```

Comment:
this looks right, you would just need to put it in ServerMain. assume you have this here for testing?